Package.swift

```diff
@@ -126,7 +126,7 @@ let package = Package(
         // library; the bootstrap scripts build the deployable version.
         .target(
             name: "PackageDescription",
-            exclude: ["CMakeLists.txt"],
+            exclude: ["CMakeLists.txt", "PackageDescription.docc"],
```

Comment:
I have not seen such usage before. Could you help point out how to reproduce the warning you mention here? I tried `swift build --product PackageDescription` and no warning was shown.